src/AaveV3NewListings_20230321/AaveV3EthNewListings_20230321.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.17;
+
+import {AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum, IEngine, Rates, EngineFlags} from 'aave-helpers/v3-config-engine/AaveV3PayloadEthereum.sol';
+
+/**
+ * @title List UNI, MKR, SNX and BAL on AaveV3Ethereum
+ * @author Llama
+ * @dev This proposal lists UNI, MKR, SNX and BAL on Aave V3 Ethereum
+ * Governance: https://governance.aave.com/t/arfc-add-uni-to-ethereum-v3/11953
+ * Snapshot UNI: https://snapshot.org/#/aave.eth/proposal/0x51d67ef69e901b34f1d111f2cd5d582c59cffa8d70b7939023febd20f7613b88
+ * Snapshot BAL: https://snapshot.org/#/aave.eth/proposal/0xe394799e4d006c15e0cb13155701de495888b7e7dad8f917a6b5dd1c8106cea5
+ * Snapshot MKR: https://snapshot.org/#/aave.eth/proposal/0xf4aec3fbab5096752be96f0e5b522f37318c1902cf8b897b049b7a94d478de73
+ * Snapshot SNX: https://snapshot.org/#/aave.eth/proposal/0x5f232a89e10d67df3aad2907e8dce3bec9708596929b3254055cf37499969b89
+ */
+contract AaveV3EthNewListings_20230321 is AaveV3PayloadEthereum {
+  address public constant UNI_PRICE_FEED = 0x553303d460EE0afB37EdFf9bE42922D8FF63220e;
+  address public constant MKR_PRICE_FEED = 0xec1D1B3b0443256cc3860e24a46F108e699484Aa;
+  address public constant SNX_PRICE_FEED = 0xDC3EA94CD0AC27d9A86C180091e7f78C683d3699;
+  address public constant BAL_PRICE_FEED = 0xdF2917806E30300537aEB49A7663062F4d1F2b5F;
```

Comment:
Should we have some test to check if these Price feeds are USD based or ETH based? I remember there being some issue around that the last time.

Reply:
double checking the feeds are correct should be good enough :)
next version of snapshot tool will also snapshot current prices to it's easier to digest if correct or not.